positions/senior-rust-cloud-developer.md

```diff
@@ -0,0 +1,78 @@
+# ☁️ Senior Rust Cloud Developer
+
+Hi, we are Enso (Y Combinator S21)! We are looking for an amazing Rust Cloud 
+developer to join our core team. We are a remote first company, working in 
+Europe and the USA.
+
+## What is Enso?
+
+**From the business perspective**, Enso is a no-code interactive data transformation
+tool. It lets you load, blend, and analyze your data, and then automate the 
+whole process, simply by connecting visual components together. It can be used for both in-memory data processing, as well as SQL analytics and transformations on modern data stack (ELT). Enso has the
+potential to disrupt the data analytics industry over the next five years. 
+Currently, the market operates using old-fashioned, limited, and non-extensible
+software which has been unable to keep up with businesses as they transition to 
+the cloud.
+
+**From a technical perspective**, Enso is a purely functional, programming 
+language with a double visual and textual syntax representation and a polyglot
+evaluation model. It means that you can mix other languages with Enso (Java, 
+JavaScript, Python, R) without wrappers and with close-to-zero performance overhead.
+
+## Who are we looking for?
+
+Enso would be a great place for you if:
+
+- You're an expert in Rust.
+- You’re any race, color, religion, gender, national origin, political affiliation, 
+  sexual orientation, marital status, disability, age.
+- You like to laugh.
+- You want to work hard, have fun doing it, and own projects from end-to-end.
+- You are friendly and like to collaborate.
+- You move fast and ask for help when needed.
+- You value being part of a team and a community.
+- You can set your ego aside because you know that good ideas can come from anywhere.
+- You enjoy working in public, putting yourself out there and showing your learning.
+- You appreciate a competitive salary and significant stock options.
+
+
+## Responsibilities
+
+- [MISSING]
```

Comment:
As a senior cloud software engineer you'll be in charge of building the SaaS offering for Enso, hosting both the language and its IDE in the cloud. This will involve:

Working closely with the internal teams to design a secure and scalable SaaS architecture.
Developing a SaaS solution based upon that design with robust tooling and reliability, as well as inbuilt support for collaboration.
Hosting the architecture on a cloud provider
Contributing to the evolution of this vibrant open-source project by bringing a new component to its ecosystem and product offering.